Description: Override python-oslo.utils dependency as it's not in trusty.
Author: Corey Bryant <corey.bryant@canonical.com>
Bug: https://bugs.launchpad.net/bugs/1432685
Forwarded: yes

--- a/nova/compute/api.py
+++ b/nova/compute/api.py
@@ -26,7 +26,6 @@
 import uuid
 
 from oslo.config import cfg
-from oslo.utils import units
 import six
 
 from nova import availability_zones
@@ -68,6 +67,7 @@
 from nova.openstack.common import log as logging
 from nova.openstack.common import strutils
 from nova.openstack.common import timeutils
+from nova.openstack.common import units
 from nova.openstack.common import uuidutils
 import nova.policy
 from nova import quota
--- a/nova/tests/compute/test_compute.py
+++ b/nova/tests/compute/test_compute.py
@@ -31,7 +31,6 @@
 import mox
 from oslo.config import cfg
 from oslo import messaging
-from oslo.utils import units
 import six
 from testtools import matchers as testtools_matchers
 
@@ -69,6 +68,7 @@
 from nova.openstack.common import jsonutils
 from nova.openstack.common import log as logging
 from nova.openstack.common import timeutils
+from nova.openstack.common import units
 from nova.openstack.common import uuidutils
 from nova import policy
 from nova import quota
